src/minimal/a-nbnbin.ads

```diff
@@ -0,0 +1,192 @@
+
```

Comment:
We should have a file header in those files.

Reply:
Fixed by https://github.com/Componolit/ada-runtime/pull/77/commits/1694bd3dfa66c1ad7748a6c6fe644599844427b6.